warcprox/warcproxy.py

```diff
@@ -179,14 +179,13 @@ def _proxy_request(self):
 
         remote_ip = self._remote_server_sock.getpeername()[0]
         timestamp = datetime.datetime.utcnow()
-
-        if warcprox_meta and 'return-capture-timestamp' in warcprox_meta:
-            return_timestamp = timestamp
-        else:
-            return_timestamp = None
+        extra_response_headers = {}
+        if warcprox_meta and 'accept' in warcprox_meta and \
+                'capture-metadata' in warcprox_meta['accept']:
+            extra_response_headers['timestamp'] = timestamp.strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
That's not quite what I meant. More like this...
```extra_response_headers['Warcprox-Meta'] = json.dumps(...)```

Reply:
I'm sorry but fully understand what you mean.

Instead of doing ``json.dumps`` inside ``mitmproxy.ProxyingRecordingHTTPResponse.begin``, 
https://github.com/internetarchive/warcprox/pull/45/files#diff-5509d710805c5f5eef9391c13093c289R176